app/src/main/res/layout/alert_dialog_title.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="20dp">
```

Comment:
![91914031_638362376986115_8497247140462985216_n](https://user-images.githubusercontent.com/483556/78005019-44a2b500-733b-11ea-8a61-4e77481e4ad5.png)

asi bych upravil ten padding..po stranach to nesedi, to zas tak nevadi, ale bottom padding bych dal urcite mensi

Reply:
Ono teda obecne ten vychozi styl dialogu je divnej, musim se na to pak podivat.